### PR TITLE
Fix run-windows app installation hang

### PR DESCRIPTION
**Pull request by ddalp**
#3190 
- "run-windows" hang is caused by waiting for user pressing Enter at end of app package installation, if a developer certificate needs to be installed. We don't see the prompt because spinner is in control. To suppress the prompt, we should always use -force parameter.
- Using -force parameter can fail if the developer mode is not enabled, this is not a problem on build machine, but can happen on fresh developer OS and it will fail when user issue "react-native run-windows" command, so adding a step to always enable developer mode.
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/3447)

### CODE REVIEW

File: vnext/docs/api/react-native-windows.iviewwindowsprops.md [hunk 1]
```diff
@@ -10,6 +10,12 @@
 export interface IViewWindowsProps extends IKeyboardProps, ViewProps 
 ```
 
+## Remarks
+
+Props type for ViewWindows component
+
+Extends: [IKeyboardProps](./react-native-windows.ikeyboardprops.md) and [react-native's ViewProps](https://facebook.github.io/react-native/docs/view)
```
Comment (ddalp) [inline vnext/docs/api/react-native-windows.iviewwindowsprops.md]: Not sure why this file is changed.. will remove.

Comment (acoates-ms) [reply vnext/docs/api/react-native-windows.iviewwindowsprops.md]: Someone should check this change in.. its auto generated, and so is probably showing up in everyones builds currently.

**Comment by acoates-ms** [comment]
An alternative would be to always allow output from the install command.  -- Which might make sense.  Its not a very noisy command output wise, compared to say build.  And would allow the user to still have proper control over the prompt.  

Not sure what peoples thoughts on that are though.

**Comment by ddalp** [comment]
I think it is still better to have spinner running by default for all steps, to be consistent. 
If there are problems, we can always ask user to run command with --logging which will turn on the prompt.

---
In reply to: [543260149](https://github.com/microsoft/react-native-windows/pull/3447#issuecomment-543260149) [](ancestors = 543260149)

File: vnext/docs/api/react-native-windows.iviewwindowsprops.md (hunk 1 above)
Comment (ddalp) [reply vnext/docs/api/react-native-windows.iviewwindowsprops.md]: I see, will leave it in the change.,

---
In reply to: [336108926](https://github.com/microsoft/react-native-windows/pull/3447#discussion_r336108926) [](ancestors = 336108926)

File: vnext/local-cli/runWindows/utils/deploy.js [hunk 2]
```diff
@@ -164,15 +164,25 @@ async function deployToDesktop(options, verbose) {
     verbose,
   );
 
+  const devmodeText = 'Enabling Developer Mode';
+  const devmodeEnable = `-ExecutionPolicy RemoteSigned Import-Module "${windowsStoreAppUtils}"; EnableDevmode "${script}"`;
```
Comment (asklar) [inline vnext/local-cli/runWindows/utils/deploy.js]: >`- [](start = 24, length = 2)

-NoProfile

File: vnext/local-cli/runWindows/utils/deploy.js [hunk 3]
```diff
@@ -164,15 +164,25 @@ async function deployToDesktop(options, verbose) {
...
+    devmodeText,
+    'powershell',
+    devmodeEnable.split(' '),
+    verbose,
+  );
+
   const installingText = 'Installing new version of the app';
-  const installApp = `-ExecutionPolicy RemoteSigned Import-Module "${windowsStoreAppUtils}"; Install-App "${script}"`;
-  const installAppCmd = options.force ? installApp + ' -Force' : installApp;
+  const installApp = `-ExecutionPolicy RemoteSigned Import-Module "${windowsStoreAppUtils}"; Install-App "${script}" -Force`;
```
Comment (asklar) [inline vnext/local-cli/runWindows/utils/deploy.js]: >installApp [](start = 8, length = 10)

add -NoProfile here too :)

File: vnext/local-cli/runWindows/utils/deploy.js (hunk 2 above)
Comment (ddalp) [reply vnext/local-cli/runWindows/utils/deploy.js]: good to know

---
In reply to: [336322464](https://github.com/microsoft/react-native-windows/pull/3447#discussion_r336322464) [](ancestors = 336322464)

**Pull request merged**